docs/TEAMS.md

```diff
@@ -0,0 +1,109 @@
+Organic Maps is led by community contributors, with functional domains occasionally establishing or dissolving their leadership structures, formal or informal. 
+
+This file outlines the functional project structure that has evolved over time. The current list of teams is available on [GitHub](https://github.com/orgs/organicmaps/teams), accessible only to [GitHub Org members](https://github.com/orgs/organicmaps/people).
+
+Below is the list of teams and groups to tag on GitHub if you need assistance in a specific area. If you are unsure which team to contact, please tag  [@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage) for help.
```

Comment:
Need to add a mention of the link between the GH teams and CODEOWNERS file and implications for reviews.. Then it'd be more clear who needs to be in the teams..

Reply:
I've added.